sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -1803,6 +1879,46 @@ public void StartRecognizeInvoicesWithWrongLocale()
             Assert.AreEqual("UnsupportedLocale", ex.ErrorCode);
         }
 
+        [RecordedTest]
+        [TestCase("1", 1)]
+        [TestCase("1-2", 2)]
+        public async Task StartRecognizeInvoicesWithOnePageArgument(string pages, int expected)
+        {
+            var client = CreateFormRecognizerClient();
+            RecognizeInvoicesOperation operation;
+
+            using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.InvoiceMultipageBlank);
+            using (Recording.DisableRequestBodyRecording())
+            {
+                operation = await client.StartRecognizeInvoicesAsync(stream, new RecognizeInvoicesOptions() { Pages = { pages } });
+            }
+
+            RecognizedFormCollection forms = await operation.WaitForCompletionAsync(PollingInterval);
+            int pageCount = forms.Sum(f => f.Pages.Count);
```

Comment:
For some reason, invoices recognition behaves differently.
- For receipts, business cards, and custom without labels, one form is returned for each specified page in the input.
- For invoices, a single form is returned, and this form contains all specified pages.

I took a look at their multipage tests and that's how they behave there as well, so I think there's nothing wrong here.

Reply:
I don't remember if we raised this concern with the service team or not. Might be good to double-check with them that the different behavior for `invoices is on purpose.
Just a message in Teams and tag Anatoly Ponomarev should do. 
also not blocking the PR

Reply:
Confirmed to be intentional.